sphinx/source/operators/start_network.rst

```diff
@@ -69,6 +69,7 @@ If the network has not yet been opened by members (see :ref:`Opening the Network
 If the network has already been opened to users, members need to trust the joining node before it can become part of the network (see :ref:`Trusting a New Node`).
 
 .. note:: When starting up the network or when joining an existing network, the network secrets required to decrypt the ledger are sealed and written to a file so that the network can later be recovered. See :ref:`Catastrophic Recovery` for more details on how to recover a crashed network.
+.. note:: By default, CCF nodes are expected to be started using IP Addresses (both IPv4 and IPV6 are supported). However it is possible to specify a domain name. In that case ``--domain=<node domain name>`` should be passed to the node at startup. Once a DNS has been setup it will then be possible to connect to the node over TLS by using the nodes domain name.
```

Comment:
Minor: Missing apostrophe, `using the nodes domain name` -> `using the node's domain name`